Src/DependencyCollector/Shared/Implementation/HttpDiagnosticSourceListener.cs

```diff
@@ -100,6 +105,81 @@ protected virtual void Dispose(bool disposing)
                 this.disposed = true;
             }
         }
+
```

Comment:
Reflection is much, much, better than dynamic!
I understand what is happening now, I completely misread it earlier and thought you already had the HttpWebRequest object and were getting properties from it.
This change is a big improvement over dynamic. :)